cmd/kubeadm/app/util/etcd/etcd.go

```diff
@@ -369,6 +370,7 @@ func (c *Client) addMember(name string, peerAddrs string, isLearner bool) ([]Mem
 		lastError   error
 		respMembers []*etcdserverpb.Member
 		learnerID   uint64
+		resp        *clientv3.MemberAddResponse
```

Comment:
I had to move `resp` out of the exponential backoff to remove the nil pointer from l.403